pom/flight_pom.py

```diff
@@ -1,72 +1,53 @@
+from conftest import *

```

Comment:
Conftest file shouldn't be imported to any file.

Reply:
removed this things sir